versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/prerequisites/preparing-the-cloud-clusters-for-registration.mdx

```diff
@@ -7,21 +7,19 @@ This topic describes how to prepare your clusters to register with KubeSlice Con
 # Authenticating Clusters 
 
 :::info
-If you have already retrieved your credentials to log in to your clusters, continue to [Labeling KubeSlice Nodes](#labeling-kubeslice-gateway-nodes)
-below.
+If you have already retrieved your credentials to log in to your clusters, continue to 
+[Labeling KubeSlice Nodes](#labeling-kubeslice-gateway-nodes) below.
 :::
 :::info 
 Perform these steps on each cluster that you want to register with the KubeSlice Controller.
 :::
 
-Before registering your clusters with KubeSlice Controller, you must
-authenticate with each cloud provider that is used in your
-installation. Each of the following commands retrieves the relevant
-`kubeconfig` and add it to your default `kubeconfig` path.
+Before registering your clusters with KubeSlice Controller, you must authenticate with each cloud provider 
+that is used in your installation. Each of the following commands retrieves the relevant`kubeconfig` and 
+add it to your default `kubeconfig` path.
 
 ## Google Kubernetes Engine
-The following information is required to retrieve your Google Kubernetes
-Engine(GKE) `kubeconfig`.
+The following information is required to retrieve your Google Kubernetes Engine(GKE) `kubeconfig`.
```

Comment:
Add space between Google Kubernetes Engine and (GKE)

Reply:
updated